autotag/main.go

```diff
@@ -14,7 +14,7 @@ import (
 type Options struct {
 	JustVersion         bool   `short:"n" description:"Just output the next version, don't autotag"`
 	Verbose             bool   `short:"v" description:"Enable verbose logging"`
-	Branch              string `short:"b" long:"branch" description:"Git branch to scan" default:"master" `
+	Branch              string `short:"b" long:"branch" description:"Git branch to scan" default:"main" `
```

Comment:
This a bit against docs:
> autotag scans the master branch for commits by default.

from https://github.com/pantheon-systems/autotag#usage 

and looks like a breaking change...

Reply:
Yep we didn't get time to finish this out.  If we want to push it forward we should update the docs, and bump the version.

Reply:
the reasoning we used for the selection was that  most folks would be moving from `master` to `main` the case where they both exist your intent is probably `main`.  If you need to specify otherwise you can with `-b`

Reply:
I meant that README should be updated.